### ISSUE TITLE
 Different results

### ISSUE THREAD

**Issue by Crazy-Child** [richard-evans/vampire-workshop#3]
Hello. Help me figure out why different results are obtained when calculating using the method 
mpirun -np 4 --oversubscribe ./vampire-parallel
and
./vampire-serial
![fig](https://github.com/user-attachments/assets/8ee0d98b-7d28-4fee-8691-71bfc3d2206c)
![fig1](https://github.com/user-attachments/assets/a584c374-de88-466b-b2f2-19b86610e267)
There is a temperature offset. Using the latest version of Vampire on Linux OS
Data from the workshop is used: https://github.com/richard-evans/vampire-workshop/tree/master/3_Higher_oder_exchange/fourspin

**Comment by richard-evans** [comment]
Yes - the pseudo-random number sequences are different for different numbers of cores. So, the results are statistically similar, but also different.
